src/ui/Icon.tsx

```diff
@@ -132,8 +132,8 @@ const Icon = ({ name, width, height, className, ...props }: Props) => {
 };
 
 Icon.defaultProps = {
-  width: 15,
-  height: 15,
+  width: 18,
+  height: 18,
 };
```

Comment:
This does affect all icons used, have you checked other pages?

Reply:
Yes it does affect all the other icons, since it's only 3px difference, it shouldn't be an issue.
But I can do an extra check

Reply:
@simon-debruijn  I did an extra check

Icons in navigation are bigger 
Icons in buttons are bigger as well... 

<img width="1378" alt="Screenshot 2022-10-26 at 10 37 21" src="https://user-images.githubusercontent.com/9402377/197977794-70b9b526-9606-4759-9a5d-83cc719d5737.png">

<img width="1192" alt="Screenshot 2022-10-26 at 10 37 47" src="https://user-images.githubusercontent.com/9402377/197977804-a4f847ef-abca-43fa-b762-baffb24d97bd.png">